source/process/release-process.md

```diff
@@ -268,23 +284,33 @@ If a bug fix release is required, run through the following steps:
 
 Once final dot release build is ready to cut:
```

Comment:
Let's rename this to `Once bug fix release is ready to cut` just so that we don't mix it with the security release.

Reply:
Done